generic/tohil.c

```diff
@@ -4508,6 +4544,7 @@ static struct PyModuleDef TohilModule = {
 int
 Tohil_Init(Tcl_Interp *interp)
 {
+    static PyThreadState *prior = NULL;
```

Comment:
Could you help me understand the use of this being `static` rather than just local?

Reply:
There is no reason.  It was an oversight.  While I don't think Tohil_Init can be reentered while it is executing, there is no need for prior to be declared static.  I probably got it by accident with some copypasta.  Fixed.